examples/with-material-ui/README.md

```diff
@@ -41,5 +32,3 @@ now
 ## The idea behind the example
 
 This example features how you use [material-ui](https://github.com/mui-org/material-ui) (Material components that implement Google's Material Design) with Next.js.
```

Comment:
Lets only leave this link, so that they can update the instructions there whenever needed and we won't have this issue coming back 👍